### ISSUE TITLE
with missing labeling

### ISSUE THREAD

**Issue by dream-mjq** [MIC-DKFZ/nnUNet#1725]
2023-10-06 11:54:29.708512: train loss : -0.3048
2023-10-06 11:54:38.296534: validation loss: -0.1194
2023-10-06 11:54:38.298707: Average global foreground Dice: 
2023-10-06 11:54:38.299034: (interpret this as an estimate for the Dice of the different classes. This is not exact.)
0 9
0 9
0 9
0 10
0 10
0 10
0 11
0 11
0 11
2023-10-06 11:54:39.287012: lr: 0.004492
2023-10-06 11:54:39.287720: This epoch took 215.674064 s

while training with my own dataset, why does my printing message have 0 9?  owing to some of my labeling missed？

**Comment by FabianIsensee** [comment]
Hey, I am not sure myself. What version of nnU-Net are you on?

**Comment by dream-mjq** [comment]
I am using the nnunet1.7.1. What's more, there is about 360 training data in imagesTr , but while my training is done, there are only two or three cases in summary.json.

**Comment by dream-mjq** [comment]
checking case xxx_001
3 3
3 3
9 9
3 3
checking case  xxx_002
3 3
3 3
9 9
3 3
checking case  xxx_003
3 3
3 3
9 9
3 3
checking case  xxx_004
3 3
3 3
9 9
3 3
checking case xxx_005
3 3
3 3
9 9
3 3
checking case Pediatric_006
3 3
3 3
9 9
3 3
checking case  xxx_007
what‘s more, I don't what does the printing mean about 33 99

**Comment by dream-mjq** [comment]
while verify_dataset_integrity, all of my cases have this printing tips
separate z, order in z is 0 order inplane is 3
separate z, order in z is 0 order inplane is 1
separate z, order in z is 0 order inplane is 1
separate z, order in z is 0 order inplane is 3
separate z, order in z is 0 order inplane is 1
separate z, order in z is 0 order inplane is 1
I do hope you can give me a guidance, thanks
